### PR TITLE
Add logback-test.xml to reduce test output

### PR DESCRIPTION
**Pull request by poornachandra**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -346,6 +352,18 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>log4j-over-slf4j</artifactId>
+        <version>${slf4j.version}</version>
+        <scope>test</scope>
```
Comment (sagarkapare) [inline pom.xml]: why is this added in `test` scope only?

**Comment by sagarkapare** [comment]
one comment else LGTM 👍

File: pom.xml (hunk 1 above)
Comment (poornachandra) [inline pom.xml]: Since we don't know what kind of logging library will be used on the cluster, we only change it for test scope.
